cluster-scope/components/centos/rolebinding.yaml

```diff
@@ -0,0 +1,15 @@
+---
+apiVersion: rbac.authorization.k8s.io/v1
+kind: RoleBinding
+metadata:
+  creationTimestamp: null
+  name: system:image-puller
+  namespace: centos
+roleRef:
+  apiGroup: rbac.authorization.k8s.io
+  kind: ClusterRole
+  name: system:image-puller
+subjects:
+- kind: ServiceAccount
+  name: builder
+  namespace: kvm-device-plugin
```

Comment:
@aleskandro could we put `'*'` in here to allow builders from all namespaces to pull the image?

Reply:
yes, it should work

Reply:
No, we can't do it this way. We could assign this role to all ServiceAccounts in the cluster, but not all the ServiceAccounts with name "builder". I'm looking for some other solution to achieve what you say.

Reply:
deferring this as not blocking